components/Footer.js

```diff
@@ -1,9 +1,128 @@
+import React from 'react';
+import { Box, Typography, Link, Paper, Divider } from '@mui/material';
+
+import '@fontsource/roboto/300.css';
+import '@fontsource/roboto/400.css';
+import '@fontsource/roboto/500.css';
+import '@fontsource/roboto/700.css';
+
+const footerPadding = 4;
+const sectionPadding = 3;
+
+const SectionHeader = ({ text }) => {
+  return (
+    <Typography
+      color='text.primary'
+      gutterBottom
+      sx={{
+        letterSpacing: 2,
+        textTransform: 'uppercase',
+        fontWeight: 'medium',
+        fontSize: 'default',
+      }}
+    >
+      {text}
+    </Typography>
+  );
+};
+
+SectionHeader.propTypes = {
+  text: PropTypes.string.isRequired,
+};
+
+const CustomLink = ({ text, link }) => {
+  return (
+    <Typography>
+      <Link
+        href={link}
+        variant='body2'
+        color='text.secondary'
+        underline='hover'
+      >
+        {text}
+      </Link>
+    </Typography>
+  );
+};
+
+CustomLink.propTypes = {
+  text: PropTypes.string.isRequired,
+  link: PropTypes.string.isRequired,
+};
+
 /**
  * A React component that is renders the footer
  * ie. the container at the very bottom of the page
- * @returns Footer
+ * @return {React.ReactElement} Footer
  */
+const Footer = () => {
+  return (
+    <Box component='footer'>
+      <Paper
+        sx={{
+          mt: 3,
+          width: '100vw',
+          backgroundColor: (theme) =>
+            theme.palette.mode === 'light'
+              ? theme.palette.grey[200]
+              : theme.palette.grey[800],
```

Comment:
```suggestion
              : theme.palette,
```


not sure if it looks nicer if the colours are kept consistent in dark mode.
But in light mode it seems that grey is nicer

Result of suggested changes
Light Mode:
<img width="314" alt="Screenshot 2023-06-11 at 11 38 38 AM" src="https://github.com/sjb-deck/deck/assets/55698556/02e1abf7-28f2-4e23-aee6-64e868bc032d">

Dark Mode:
<img width="316" alt="Screenshot 2023-06-11 at 11 39 16 AM" src="https://github.com/sjb-deck/deck/assets/55698556/c3113375-5a88-4361-b389-35d08ff95ff0">